indico/core/db/sqlalchemy/review_questions.py

```diff
@@ -111,3 +135,20 @@ def get_review_rating(self, review, allow_create=False):
             rating_class = type(self).ratings.prop.mapper.class_
             rating = rating_class(question=self, review=review)
         return rating
+
+    @property
+    def title(self):
+        return self.text
+
+    @property
+    def description(self):
+        """Required by BaseField for creating an instance of WTForms field"""
+        return None
+
+    @property
+    def no_score(self):
+        return self.field_type != 'rating' or self._no_score
+
```

Comment:
You might want to add a `.expression` for the property; like this you can use it in `filter()` criteria as well.